### ISSUE TITLE
Updating mods crashes ckan, modulemanager error

### ISSUE THREAD

**Issue by Lemaya** [KSP-CKAN/CKAN#1154]
Updating mods crashes ckan

ModuleManager 2.6.5 is installed but not managed by ckan

```
The following inconsistencies were found:
ModuleManager requires a version between 2.6.3 and any version inclusive.. However a incompatible version, autodetected dll, is already installed   bei CKAN.RelationshipResolver.ResolveStanza(IEnumerable`1 stanza, SelectionReason reason, RelationshipResolverOptions options, Boolean soft_resolve)
   bei CKAN.RelationshipResolver.Resolve(CkanModule module, RelationshipResolverOptions options)
   bei CKAN.RelationshipResolver.AddModulesToInstall(IEnumerable`1 modules)
   bei CKAN.ModuleInstaller.Upgrade(IEnumerable`1 identifiers, NetAsyncDownloader netAsyncDownloader)
   bei CKAN.Main.InstallMods(Object sender, DoWorkEventArgs e)
   bei System.ComponentModel.BackgroundWorker.WorkerThreadStart(Object argument)


************** Geladene Assemblys **************
mscorlib
    Assembly-Version: 4.0.0.0.
    Win32-Version: 4.0.30319.34209 built by: FX452RTMGDR.
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll.
----------------------------------------
ckan
    Assembly-Version: 0.0.0.0.
    Win32-Version: 0.0.0.0.
    CodeBase: file:///C:/Games/KerbalSpaceProgram/ckan.exe.
----------------------------------------
System.Windows.Forms
    Assembly-Version: 4.0.0.0.
    Win32-Version: 4.0.30319.34251 built by: FX452RTMGDR.
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll.
----------------------------------------
System.Drawing
    Assembly-Version: 4.0.0.0.
    Win32-Version: 4.0.30319.34209 built by: FX452RTMGDR.
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll.
----------------------------------------
System
    Assembly-Version: 4.0.0.0.
    Win32-Version: 4.0.30319.34238 built by: FX452RTMGDR.
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll.
```

**Comment by Postremus** [comment]
what version of ckan do you use?

**Comment by Lemaya** [comment]
Just autoupdated 1.8.2-g0ff265

**Comment by Lemaya** [comment]
I tried to work around it by letting ckan manage ModuleManager, butI dont know how i get ckan to manage ModuleManager ( i once installed it manually).

Deleting the files still shows the autodetected.dll in ckan and I dont know if there is a config file to change

**Comment by Lemaya** [comment]
I can update mods, which dont depend on ModuleManager

**Comment by pjf** [comment]
Drat. Autodetected mods don't have a version, so our code considers them to never satisfy version requirements. Really we should consider them to *always* satisfy requirements (although the eventual plan is to get rid of them entirely: #949 ).

Let me see what I can do about getting a fix out this arvo.

**Comment by pjf** [comment]
@Lemaya : Out of curiosity, are you updating mods via the command line or the GUI? Users shouldn't be seeing that level of exposed messages, so that's potentially a second bug that's been exposed and which we should also be fixing.

**Comment by RichardLake** [comment]
@Lemaya 
I think you can convert it to ckan managed via the commandline. Remove all of the modulemanager dlls and run ckan.exe install ModuleManager

**Comment by pjf** [comment]
@Lemaya : This should now be fixed in v1.8.3 (just released). Thank you so much for the bug report!

**Comment by Lemaya** [comment]
I am using the GUI i am not so good with the command line.

Thanks alot :)
